pkg/lib/long-running-process.js

```diff
@@ -82,7 +82,7 @@ export class LongRunningProcess {
     /*  Stop long-running process while it is RUNNING, or reset a FAILED one */
     terminate() {
         if (this.state !== ProcessState.RUNNING && this.state !== ProcessState.FAILED)
-            throw new Error(`cannot terminate LongRunningProcess in state ${this.sate}`);
+            throw new Error(`cannot terminate LongRunningProcess in state ${this.state}`);
```

Comment:
This is an actual code change. I left it in as it looks like it fixes a bug. Please review this to make sure it is actually supposed to be state and not "sate".

I think it is supposed to be state, as it checks for `this.state`, so I'm pretty sure codespell spotted and fixed a bug here.

Reply:
That's correct, thanks!